### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by daniel-farina**
## Description

This PR adds a `security.md` file to the repository, which outlines the vulnerability disclosure process and how to report security vulnerabilities in Osmosis. The document provides clear instructions on how to report vulnerabilities, what to expect during the disclosure process, and our commitment to keeping Osmosis and its users safe and secure.

## Changes

- Added `security.md` file to the repository

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Standard Priority Bug
+
+For a bug that is non-sensitive and/or operational in nature rather than a critical vulnerability, please add it as a GitHub issue.
+
+If it is not triaged in a couple of days, feel free to tag @p0mvn, @czarcas7ic, @mattverse, @daniel-farina, @ValarDragon
+
+## Critical Bug or Security Issue
+
+If you're here because you're trying to figure out how to notify us of a security issue, please send us a message on https://security.osmosis.zone/ or via [Telegram](https://t.me/OsmoSec)
```
Comment (czarcas7ic) [inline SECURITY.md:19]: I looked at the form here. It looks good except the "who reported the bug" field feels odd (has options as support, sales, customer, internal)

**Review by czarcas7ic — APPROVED** [review]
LGTM, glad we have this now :)

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+For a bug that is non-sensitive and/or operational in nature rather than a critical vulnerability, please add it as a GitHub issue.
+
+If it is not triaged in a couple of days, feel free to tag @p0mvn, @czarcas7ic, @mattverse, @daniel-farina, @ValarDragon
+
+## Critical Bug or Security Issue
+
+If you're here because you're trying to figure out how to notify us of a security issue, please send us a message on https://security.osmosis.zone/ or via [Telegram](https://t.me/OsmoSec) 
+
+Please avoid opening public issues on GitHub that contain information about a potential security vulnerability as this makes it difficult to reduce the impact and harm of valid security issues.
```
Comment (ValarDragon) [inline SECURITY.md:21]: Should they also be pointed to the validators security discussion chat?

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Osmosis Bug Reporting and Feature Requests
+
+Osmosis uses GitHub to manage feature requests and bugs for its platform. This is done via GitHub Issues.
+
+## Feature Request
+
+For a feature request, please [create a GitHub issue](https://github.com/osmosis-labs/osmosis/issues/new). Clearly state your use case and what value it will bring to other users or developers on Osmosis.
+
+If it is something that can be handled by a parameter change, discuss it on [the discussions](https://github.com/osmosis-labs/osmosis/discussions) or on [Discord](https://discord.com/invite/osmosis) in `#governance-proposals` and consider a governance proposal.
```
Comment (ValarDragon) [inline SECURITY.md:9]: Should these be on github or commmonwealth? cc (cc @johnnywyles)

**Review by p0mvn — APPROVED** [review]

**Comment by czarcas7ic** [comment]
@daniel-farina are you able to resolve comments and then we can get this merged?

File: SECURITY.md (hunk 1 above)
Comment (daniel-farina) [reply SECURITY.md:19]: This is not supposed to be on the form. Removed 👍

File: SECURITY.md (hunk 2 above)
Comment (daniel-farina) [reply SECURITY.md:21]: what chat are you referring to? Ideally we use this form as a filter between the reporters and wider leaks.

File: SECURITY.md (hunk 3 above)
Comment (daniel-farina) [reply SECURITY.md]: > @daniel-farina are you able to resolve comments and then we can get this merged?

This one got lost in time. Solving comments and merging 👍

Comment (daniel-farina) [reply SECURITY.md]: updated 👍

**Comment by daniel-farina** [comment]
> @daniel-farina are you able to resolve comments and then we can get this merged?

This one got lost in time. Solving comments and merging 👍

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+For a feature request, please [create a GitHub issue](https://github.com/osmosis-labs/osmosis/issues/new). Clearly state your use case and what value it will bring to other users or developers on Osmosis.
+
+If it is something that can be handled by a parameter change, we encourage you to join the Osmosis Commonwealth Forum for discussions. You can participate in the forum by visiting https://gov.osmosis.zone/overview. Additionally, you can also join the Osmosis Discord community at https://discord.com/invite/osmosis and engage in the #governance-proposals channel to discuss and consider governance proposals.
+
+## Standard Priority Bug
+
+For a bug that is non-sensitive and/or operational in nature rather than a critical vulnerability, please add it as a GitHub issue.
+
+If it is not triaged in a couple of days, feel free to tag @p0mvn, @czarcas7ic, @mattverse, @daniel-farina, @ValarDragon
```
Comment (AlpinYukseloglu) [inline SECURITY.md:15]:
```suggestion
If it is not triaged in a couple of days, feel free to tag @p0mvn, @czarcas7ic, @mattverse, @daniel-farina, @ValarDragon, @AlpinYukseloglu 
```

**Review by AlpinYukseloglu — APPROVED** [review]
Looks good, thanks for adding this!

**Comment by czarcas7ic** [comment]
Hey @daniel-farina, bumping this

**Comment by daniel-farina** [comment]
> 

ops, all resolved.

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+We ask security researchers to keep vulnerabilities and communications around vulnerability submissions private and confidential until a patch is developed. In addition to this, we ask that you:
+
+- Allow us a reasonable amount of time to correct or address security vulnerabilities.
+- Avoid exploiting any vulnerabilities that you discover.
+- Demonstrate good faith by not disrupting or degrading Osmosis's services.
+
+## Vulnerability Disclosure Process
+
+1. **Report:** Report the vulnerability by sending a message to https://security.osmosis.zone/
```
Comment (ValarDragon) [inline SECURITY.md:33]: Does anyone check this? I'd rather keep messaging clear to Telegram if thats the only thing thats going to be checked

Comment (AlpinYukseloglu) [reply SECURITY.md:33]: +1

**Pull request closed without merging**